os_installer2/pages/users.py

```diff
@@ -20,11 +20,13 @@
 DATA_COLUMN = 1
 
 
-class UserPanel(Gtk.VBox):
+class UserPanel(Gtk.Box):
     """Userpanel. Represents a user. Whoda thunk it. """
 
     def __init__(self, user):
-        Gtk.VBox.__init__(self)
+        Gtk.Box.__init__(self)
```

Comment:
`Gtk.Box.__init__(self, orientation=Gtk.Orientation.VERTICAL, spacing=0)` should also work?